README.md

```diff
@@ -32,22 +33,237 @@ scheduled_task { 'Run Notepad':
+  }],
+}
+~~~
+
+If we need to have the cleanup script run every night we can use a daily trigger.
+Just changing the trigger schedule from `once` to `daily` will do the trick.
+
+~~~ puppet
+scheduled_task { 'Disk Cleanup Nightly':
+  command   => 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe',
+  arguments => '-File "C:\\Scripts\\Clear-DiskSpace.ps1",
+  enabled   => 'true',
+  trigger   => [{
+    schedule         => 'daily',
+    start_time       => '23:20'
+  }],
+}
+~~~
+
+You can also set your scheduled tasks to repeat during a set time block.
+Re-using the idea of a cleanup script, this scheduled task begins at the same time every day and runs once an hour from seven in the morning to seven at night as the SYSTEM account.
+
+~~~ puppet
+scheduled_task { 'Disk Cleanup Daily Repeating':
+  ensure    => 'present',
+  command   => 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe',
+  arguments => '-File "C:\\Scripts\\Clear-DiskSpace.ps1",
+  enabled   => 'true',
+  trigger   => [{
+    'schedule'         => 'daily',
+    'start_time'       => '07:00',
+    'minutes_duration' => '720',   # Specifies the length of time, in minutes, the task is active
+    'minutes_interval' => '60'     # Causes the task to run every hour
+  }],
+  user      => 'system',             # Specifies the account to run the task as
+}
+~~~
+
+The downside to that task is that it causes the cleanup script to run _every_ day, even on the weekends when there is no activity.
+We can instead use a weekly trigger to fix this:
+
+~~~puppet
+scheduled_task { 'Disk Cleanup Weekly Repeating':
+  ensure    => 'present',
+  command   => 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe',
+  arguments => '-File "C:\\Scripts\\Clear-DiskSpace.ps1",
+  enabled   => 'true',
+  trigger   => [{
+    'schedule'         => 'weekly',
+    'start_time'       => '07:00',
+    'day_of_week'      => ['mon', 'tues', 'wed', 'thurs', 'fri'], # Note the absence of Sunday and Monday
+    'minutes_interval' => '60',
+    'minutes_duration' => '720'
+  }],
+  user      => 'system',
+}
+~~~
+
+Similarly, we can schedule our cleanup script to run monthly if we decide the cleanup script doesn't need to run as often or is particularly resource intensive.
+The following example sets the scheduled task to run at 0700 on the first day of the month every month:
+
+~~~puppet
+scheduled_task { 'Disk Cleanup Monthly First Day':
+  ensure    => 'present',
+  command   => 'C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe',
+  arguments => '-File "C:\\Scripts\\Clear-DiskSpace.ps1",
+  enabled   => 'true',
+  trigger   => [{
+    'schedule'         => 'monthly',
+    'start_time'       => '07:00',
+    'on'               => [1]        # Run every month on the first day of the month.
+  }],
+  user      => 'system',
+}
+~~~
+
+One problem with the monthly trigger above is that we can't guarantee that the cleanup script doesn't run during a work day.
```

Comment:
Double negative here makes it a bit harder to read.  Not sure what to reword to.

Reply:
> With the monthly trigger above there is no guarantee that the first day of the month is on a weekend.
> This means that there's a reasonable chance that the script will execute during working hours and impact productivity.

This is a possible rewrite to clarify the problem. It's longer but explains the actual impact a bit more.